shap/plots/_utils.py

```diff
@@ -226,3 +239,40 @@ def sort_inds(partition_tree, leaf_values, pos=None, inds=None):
     sort_inds(partition_tree, leaf_values, right, inds)
 
     return inds
+
+
+# Various ways to specify a desired axis limit in plots
+AxisLimitSpec = Union[Explanation, str, float, None]
```

Comment:
Now that we have `typing_extensions` as a dependency, we could use `TypeAlias` here to make it explicit that this is an Alias and not a regular variable assignment.

```suggestion
AxisLimitSpec: TypeAlias = Union[Explanation, str, float, None]
```